spec/controllers/partners_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe PartnersController, type: :controller do
```

Comment:
We've been migrating our controller specs over to request specs (I forget the reason, but I suspect DHH was involved). 

If you wouldn't mind adding this to the `spec/requests/partner_requests_spec.rb`, that'd be awesome :) -- the Request specs focus entirely on "does the action work and return success?"; if you wanted to also create a system spec (to follow the UI flow) that's cool too -- LMK if you need an extra hand in writing those. You can probably copy-paste what's there and just modify it. `accept_confirm` would handle the confirmation dialog.

Reply:
@armahillo Thanks for the feedback! We moved the tests to `spec/requests/partner_requests_spec.rb` but we are not sure if they are testing the right thing. We'd like to also test the UI. Where should we put those specs?